src/attngan/code/main.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import print_function
+
+from miscc.config import cfg, cfg_from_file
+from datasets import TextDataset
+from trainer import condGANTrainer as trainer
+
+import os
+import sys
+import time
+import random
+import pprint
+import datetime
+import dateutil.tz
+import argparse
+import numpy as np
+
+import torch
+import torchvision.transforms as transforms
+
+os.environ['KMP_DUPLICATE_LIB_OK']='True'
+
+dir_path = (os.path.abspath(os.path.join(os.path.realpath(__file__), './.')))
```

Comment:
Would recommend use `PYTHONPATH` in this case

Reply:
will explore `PYTHONPATH` for future projects. Thanks for the pro tip!